doc/release/release_dev.rst

```diff
@@ -52,6 +52,10 @@ API Changes
 - Default value of ``order`` parameter has been set to ``rc`` in
   ``skimage.feature.hessian_matrix``.
 - ``skimage.util.img_as_*`` functions no longer raise precision and/or loss warnings.
+- In `skimage.draw.ellipse`, the argument ``rotation`` will be replaced with
+``orientation`` in version 0.19.
+- In `skimage.draw.ellipse`, the argument ``orientation`` will come before
+the argument ``shape`` in version 0.19.
```

Comment:
Why all the way to 0.19? Apologies if I've missed some discussion around this.

Reply:
I guess it's going to take a while to implement this correctly, @jni. That's why I tried to estimate a longer deprecation path (one version more than the usual).